Automattic/wp-calypso
# DIFM: Removes site info collection step

#### Changes proposed in this Pull Request

* This is a follow up to #62122. Since the new steps `difm-options` and `social-profiles` have been tested, we can completely replace the `site-info-collection` step with the new steps.

#### Testing instructions



* Start the flow at `/start/do-it-for-me`. (Feature flag not required)
* Complete the flow and ensure that there are no JS errors.
